packages/react/src/components/ValueCard/Attribute.jsx

```diff
@@ -37,9 +38,9 @@ const propTypes = {
   secondaryValue: PropTypes.shape({
     color: PropTypes.string,
     trend: PropTypes.oneOf(['up', 'down']),
-    value: PropTypes.any,
```

Comment:
There could be an argument for keeping any here and adding an exception, but I added all the various types used currently.